lib/recurly/item.rb

```diff
@@ -26,8 +26,7 @@ def changed_attributes
     end
 
     def reactivate
-      return false unless link? :reactivate
-      reload follow_link :reactivate
+      reload API.put("#{uri}/reactivate")
```

Comment:
Can we add a comment to the top of this method that this should be changed to use a link (when it's added on the server)?